linera-rpc/src/grpc_network.rs

```diff
@@ -559,29 +560,85 @@ impl ValidatorNode for GrpcClient {
         &mut self,
         query: linera_core::data_types::ChainInfoQuery,
     ) -> Result<linera_core::data_types::ChainInfoResponse, NodeError> {
-        debug!(query = ?query);
+        debug!(?query);
         client_delegate!(self, handle_chain_info_query, query)
     }
 
     #[instrument(target = "grpc_client", skip_all, err, fields(address = self.address))]
     async fn subscribe(&mut self, chains: Vec<ChainId>) -> Result<NotificationStream, NodeError> {
+        const MIN_DELAY: u64 = 1;
+        const MAX_DELAY: u64 = 10;
+        const DELAY_INCREMENT: u64 = 1;
```

Comment:
We should probably make these configurable, but I'm not sure what's better:

* We could just have a configurable `delay` and `max_retries`, and not increase the delay between retries.
* Alternatively, we could have a `min_delay`, `max_delay` and `delay_increment`.

Reply:
I've used `max_retries` + `delay` (used an increment) in several places

Reply:
Done.